docs/source/user-guide/nlp/coming_soon.rst

```diff
@@ -0,0 +1,11 @@
+.. _coming_soon:
+
+=================================
+Deepchecks for NLP - Coming Soon!
+=================================
+
+We are working on new and exciting checks for the new ``deepchecks.nlp``. Stay tuned!
+
+In the meanwhile, you can read about the supported task types in this
+:doc:`guide </user-guide/nlp/supported_tasks>`, and see some of the currently implemented checks at our
+:doc:`API ref <../../../api/deepchecks.nlp>`.
```

Comment:
better to change this one to a non-relational link, does </api/deepchecks.nlp> not work?
(if not I guess maybe it doesn't work since it's not in the index and then we should just change this later on...)